bin/config_py.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import click
+import os
+import sys
+import shutil
+
+
+CONF_DIR_NAME = 'config'
+DEV_FILE = 'config-dev.py'
+ERROR_ALREADY_EXISTS_FMT = '"config" module already exists for the {} module'
+
+
+def generate_root_config():
+    src_dir = sys.path[0] + '/../fixtures/root/'
```

Comment:
It should be all done via `os.path.join` - we want to run on Windows, don't we/

Reply:
good call